hiob-ludolf-centre-for-ethiopian-studies.csl

```diff
@@ -177,13 +177,13 @@
       <if variable="event">
         <choose>
           <if variable="genre" match="none">
-            <text term="presented at" text-case="capitalize-first" suffix=" "/>
+            <text term="presented at" text-case="lowercase" suffix=" "/>
```

Comment:
thanks, done!

Quoting "Rintze M. Zelle" <notifications@github.com>:

>> @@ -177,13 +177,13 @@
>>        <if variable="event">
>>          <choose>
>>            <if variable="genre" match="none">
>> -            <text term="presented at" text-case="capitalize-first"  
>> suffix=" "/>
>> +            <text term="presented at" text-case="lowercase" suffix=" "/>
>
> @eu-genia, with very few exceptions, English terms are defined in  
> lowercase, so `text-case="lowercase"` is usually redundant and can  
> be omitted.
>
> ---
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/citation-style-language/styles/pull/1989/files/4e6138152e694f3cdddbd958a92c49fcb1efd12b#r57949768



-- 
Eugenia Sokolinski
Project Coordinator
Hiob Ludolf Centre for Ethiopian Studies
Asien Afrika Institut
Universit�t Hamburg
Alsterterrasse 1
20354 Hamburg

eugenia.sokolinski@uni-hamburg.de

http://www1.uni-hamburg.de/ethiostudies
http://www.traces.uni-hamburg.de